src/OpenRiaServices.Server.EntityFrameworkCore/Framework/OpenRiaServices.Server.EntityFrameworkCore.csproj

```diff
@@ -10,8 +10,13 @@
     <PackageLicenseFile>LICENSE.md</PackageLicenseFile>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="3.1.24" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="3.1.24" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Condition="'$(TargetFramework)'=='netstandard2.0'" Version="3.1.28" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Condition="'$(TargetFramework)'=='netstandard2.0'" Version="3.1.28" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Condition="'$(TargetFramework)'=='netstandard2.0'" Version="3.1.28" />
+
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Condition="'$(TargetFramework)'!='netstandard2.0'" Version="6.0.0" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Condition="'$(TargetFramework)'!='netstandard2.0'" Version="6.0.0" />
```

Comment:
Is the design package required to build?
I would prefer that we avoid adding nuget dependencies if possible.

Reply:
Not needed. Could remove it and in still builds.